consensus/replay.go

```diff
@@ -137,7 +137,7 @@ func (cs *State) catchupReplay(csHeight int64) error {
 	}
 	defer gr.Close()
 
-	cs.Logger.Info("Catchup by replaying consensus messages", "height", csHeight)
+	cs.Logger.Debug("Catchup by replaying consensus messages", "height", csHeight)
```

Comment:
This again is a message that should be printed only once, Info level looks more appropriate.